README.md

```diff
@@ -1 +1,54 @@
-# Leaderboard-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard App
+
+>I am setting up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I have developped a first working version of the app following a wireframe, but with minimum styling just focusing on the functionality.
+
+
+## Built With
+
+- Html
+- Css
+- npm
+- webpack
+- javascript
+
+## Webpack server
+
+[Server link](http://localhost:8080/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- VS Code
+- Chrome Browser
+- Gitbash
+
+
+### Usage
+- `$ git clone https://github.com/Kingstalux/Leaderboard-App`
+- `$ cd leaderboard-app`
+- run the command `code .` to open the file in VS Code
+- run the command `Shift+Alt+R` to open the file in the browser
```

Comment:
- [NOT REALLY OPTIONAL] Your getting started needs a little bit of improvement. Please, ensure you address the following before submitting tasks for the next miletone:
    - This is a `webpack` project that has dependencies that must be installed before one can get started with it. Please instruct the user on how to do this.
    - Commands for building and also starting the project must also be listed here. Please work on this too.

Reply:
okay thanks, I'll implement that in the next branch